databricks_cli/unity_catalog/cred_cli.py

```diff
@@ -0,0 +1,247 @@
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def delete_credential_cli(api_client, name):
+    """
+    Delete a storage credential.
+
+    Calls the 'deleteStorageCredential' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    UnityCatalogApi(api_client).delete_storage_credential(name)
+
+
+#############  Data Access Configuration Commands  ############
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Create data access configuration.')
+@click.option('--metastore-id', required=True, type=MetastoreIdClickType(),
+              help='Unique identifier of the metastore parent of the DAC.')
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to POST.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/unity-catalog/data-access-configurations'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def create_dac_cli(api_client, metastore_id, json_file, json):
+    """
+    Create new data access configuration.
+
+    Calls the 'createDataAccessConfiguration' RPC endpoint of the Unity Catalog service.
+    The specification for the request json can be found at <insert doc link here>.
+    Returns the properties of the newly-created DAC.
+
+    """
+    json_cli_base(json_file, json,
+                  lambda json: UnityCatalogApi(api_client).create_dac(metastore_id, json),
+                  encode_utf8=True)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='List data access configurations.')
+@click.option('--metastore-id', required=True, type=MetastoreIdClickType(),
+              help='Unique identifier of the metastore parent of the DAC(s).')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_dacs_cli(api_client, metastore_id):
+    """
+    List data access configurations.
+
+    Calls the 'listDataAccessConfigurations' RPC endpoint of the Unity Catalog service.
+    Returns array of DataAccessConfigurations.
+
+    """
+    dacs_json = UnityCatalogApi(api_client).list_dacs(metastore_id)
+    click.echo(mc_pretty_format(dacs_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get data access configuration.')
+@click.option('--metastore-id', required=True, type=MetastoreIdClickType(),
+              help='Unique identifier of the metastore parent of the DAC.')
+@click.option('--dac-id', required=True, type=DacIdClickType(),
+              help='Data access configuration ID.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_dac_cli(api_client, metastore_id, dac_id):
+    """
+    Get data access configuration details.
+
+    Calls the 'getDataAccessConfiguration' RPC endpoint of the Unity Catalog service.
+    Returns details of the DAC specified by its id (TODO: lookup by DAC name?).
```

Comment:
please address TODO

Reply:
We're wanting to deprecate DACs, so I wouldn't spend time on this.

Reply:
DACs are removed in #471. Also see https://github.com/databricks/databricks-cli/pull/471#discussion_r896919041.